checkstyle.xml

```diff
@@ -184,10 +184,6 @@
             <property name="lineWrappingIndentation" value="8"/>
             <property name="arrayInitIndent" value="4"/>
         </module>
-        <module name="AbbreviationAsWordInName">
-            <property name="ignoreFinal" value="false"/>
```

Comment:
What does this do?

Reply:
specifies that the rule ignores final variables, eg `MY_CONSTANT` http://checkstyle.sourceforge.net/config_naming.html#AbbreviationAsWordInName